### PR TITLE
Remove repo name

### PR DESCRIPTION
**Pull request by MooreGuy**
Now that repo is the new column that is being used in all of the tables, the old repo_name and repo_owner columns can be dropped for good.

QA: Run the migrations, then test all the normal features of pulldasher, signing off and having it update, moving pulls between columns, and test all the links to pulls and their build statuses

Closes: https://github.com/iFixit/pulldasher/issues/79

### CODE REVIEW

**Comment by MooreGuy** [comment]
dev_block :+1: on https://github.com/iFixit/pulldasher/pull/82 being merged

**Comment by kwiens** [comment]
#82 is merged now — unblock?

**Comment by MooreGuy** [comment]
Yeah, time to get the merge conflicts fixed.

**Comment by danielbeardsley** [comment]
Maybe rebase so there is only a few commits?

**Comment by MooreGuy** [comment]
I'll get to this soon, it's just not a critical pull compared to the other stuff that needs to be done at the moment.

File: app.js [hunk 1]
```diff
@@ -55,18 +55,20 @@ authManager.setupRoutes(app);
...
-})
-// Get the most recent version of each pull from the API
-.then(function () {
-   return refresh.openPulls();
-}).done();
+config.repos.forEach(function(repo) {
+   // Load open pulls from the DB so we don't start blank.
+   dbManager.getOpenPulls(repo).then(function(pulls) {
+      pullQueue.pause();
+      pulls.forEach(function(pull) {
```
Comment (sterlinghirsh) [inline app.js]: Is there a reason not to do pulls.forEach(pullManager.updatePull)?

**Comment by sterlinghirsh** [comment]
CR :zap:

**Comment by kwiens** [comment]
Pøke

**Comment by danielbeardsley** [comment]
This might need a rebase... the commit history looks bigger than it should. Probably should only have the last 5 commits or so.

**Comment by MooreGuy** [comment]
un_dev_block :+1: did the rebase

**Comment by lexahall** [comment]
@MooreGuy I can QA this with you.

**Comment by djmetzle** [comment]
CR :+1: 
I was going to say, perhaps move this migration: https://github.com/iFixit/pulldasher/pull/83/commits/385e52537ab16abac918ea6b7b6d315b6eb35c84
to a separate pull, but this is pulldasher, so it's probably fine. I'm going to take a stab in the dark and guess that migrations get ran on startup, right? If not, then it seems like that migration could be a break change under a live running pulldasher. Nothing to block on.

File: models/pull.js [hunk 2]
```diff
@@ -223,8 +223,7 @@ Pull.getFromDB = function(data, signatures, comments, commitStatus, labels) {
          repo: {
             owner: {
                login: data.repo_owner
-            },
-            name: data.repo_name,
+            }
```
Comment (danielbeardsley) [inline models/pull.js]: We may need to adjust these:
```
public/js/Pull.js:8:         return 'https://github.com/' + this.head.repo.owner.login + '/' + this.head.repo.name + '/pull/' + this.number;
views/standard/spec/pageIndicators.js:105:            var repo = frozen[0].head.repo.name;
```

**Comment by danielbeardsley** [comment]
dev_block 👍  on a comment.

**Comment by MooreGuy** [comment]
@djmetzle The migrations for pulldasher need to be run manually. How we have been doing the migrations is run the migration and then restart pulldasher so it is running the new code. Doing it this way won't break anything.

**Comment by MooreGuy** [comment]
Actually I take that back, I didn't realize there were two migrations @djmetzle I'm going to split them up.

**Comment by djmetzle** [comment]
@MooreGuy I wouldn't worry too much then. You can just stop the pulldasher service and restart it after the migrations. I was just concerned that if pulldasher was running when you ran it, that migration could break stuff.

**Comment by MooreGuy** [comment]
dev_block on this having the second query/schema change removed

**Comment by MooreGuy** [comment]
un_dev_block :+1:

**Comment by MooreGuy** [comment]
QA :+1: I checked the frozen pulls link, the repo links, and CR'd QA'd stuff

**Comment by colebemis** [comment]
CR 👍

**Comment by danielj41** [comment]
CR 👍

**Comment by MooreGuy** [comment]
CR :+1:

**Comment by mlahargou** [comment]
Guy do you want to give this a QA?

**Comment by danielbeardsley** [comment]
CR 👍

**Comment by MooreGuy** [comment]
@mlahargou You should be set up to test it now, I updated QA instructions to follow. There isn't a whole lot in pulldasher, so it's pretty reasonable to skim through most of its functionality.

**Comment by MooreGuy** [comment]
Actually yeah, I already went through this, I'll do it

**Comment by lexahall** [comment]
@MooreGuy I'm taking the external block off. Feel free to QA it if you want, but it seems like someone else can do this.

**Comment by MooreGuy** [comment]
I got really frustrated with dealing with two pull requests that both need to be merged to QA, so this one plus more is now in: https://github.com/iFixit/pulldasher/pull/109

**Pull request closed without merging**